std/haxe/macro/Printer.hx

```diff
@@ -138,7 +138,7 @@ class Printer {
 			case TParent(ct): "(" + printComplexType(ct) + ")";
 			case TOptional(ct): "?" + printComplexType(ct);
 			case TNamed(n, ct): n + ":" + printComplexType(ct);
-			case TExtend(tpl, fields): '{> ${tpl.map(printTypePath).join(" >, ")}, ${fields.map(printField).join(", ")} }';
+			case TExtend(tpl, fields): '{> ${tpl.map(printTypePath).join(" >, ")}, ${[for (f in fields) printField(f) + "; "].join("")}}';
```

Comment:
The field fix is fine, but what's up with the `.join(" >, ")`? Surely that should be `.join(", > ")`…

```suggestion
			case TExtend(tpl, fields): '{> ${tpl.map(printTypePath).join(", > ")}, ${[for (f in fields) printField(f) + "; "].join("")}}';
```

Reply:
This syntax requires a trailing `,` IIRC.

Reply:
Good catch @Aurel300.
@Gama11 trailing comma ensured after `join`, before fields.